pom.xml

```diff
@@ -120,7 +120,6 @@
         <com.google.apis.client.version>1.26.0</com.google.apis.client.version>
         <com.google.apis.compute.version>v1-rev20190607-${com.google.apis.client.version}</com.google.apis.compute.version>
         <com.google.apis.storage.version>v1-rev20190523-${com.google.apis.client.version}</com.google.apis.storage.version>
-        <jdk.surefire.argLine><!-- empty placeholder --></jdk.surefire.argLine>
```

Comment:
I see CI failures due to this being missing:

```
Error: Could not find or load main class ${jdk.surefire.argLine}
```

Can you keep this placeholder here, and then remove the specific lines that are no longer needed from the `java-9+` profile?

We do still need some of the lines, due to datasketches. The readme at https://github.com/apache/datasketches-memory mentions that some `--add-opens` and `--add-exports` are needed to work with direct memory and memory maps on JDK 11. And we need the empty placeholder for JDK 8.

Reply:
@gianm All checks have passed

Reply:
thanks!